.github/workflows/laravel-package.yml

```diff
@@ -0,0 +1,20 @@
+name: Laravel
+
+on: [push]
```

Comment:
We need to be able to do the same thing we're doing with TravisCI and deal with these builds on PRs as well. Push may be a little excessive for this repo, but I'm fine leaving it as is for now.